packages/loader/container-loader/src/serializedStateManager.ts

```diff
@@ -215,3 +142,68 @@ export class SerializedStateManager {
 		);
 	}
 }
+
+export async function fetchISnapshot(
+	mc: MonitoringContext,
+	storageAdapter: Pick<IDocumentStorageService, "getSnapshot">,
+	specifiedVersion: string | undefined,
+): Promise<{ snapshot?: ISnapshot | ISnapshotTree; version?: IVersion }> {
+	const snapshot =
+		(await storageAdapter.getSnapshot?.({
+			versionId: specifiedVersion,
+		})) ?? undefined;
```

Comment:
```suggestion
		await storageAdapter.getSnapshot?.({
			versionId: specifiedVersion,
		});
```

Reply:
this is probably pre-exisitng, but there isn't a need to do `undefined ?? undefined`